code/modules/research/machinery/_production.dm

```diff
@@ -4,28 +4,27 @@
 	name = "technology fabricator"
 	desc = "Makes researched and prototype items with materials and energy."
 	layer = BELOW_OBJ_LAYER
-	var/consoleless_interface = FALSE			//Whether it can be used without a console.
-	var/efficiency_coeff = 1				//Materials needed / coeff = actual.
+	var/consoleless_interface = FALSE			// Whether it can be used without a console.
+	var/efficiency_coeff = 1				// Materials needed / coeff = actual.
 	var/list/categories = list()
 	var/datum/component/remote_materials/materials
 	var/allowed_department_flags = ALL
-	var/production_animation				//What's flick()'d on print.
+	var/production_animation				// What's flick()'d on print.
 	var/allowed_buildtypes = NONE
 	var/list/datum/design/cached_designs
 	var/list/datum/design/matching_designs
-	var/department_tag = "Unidentified"			//used for material distribution among other things.
+	var/department_tag = "Unidentified"			// Used for material distribution among other things.
 	var/datum/techweb/stored_research
 	var/datum/techweb/host_research
 
 	var/search = null
 	var/selected_category = null
 
-	/// What color is this machine's stripe? Leave null to not have a stripe.
-	var/stripe_color = null
+	var/stripe_color = null	// What color is this machine's stripe? Leave null to not have a stripe.
```

Comment:
Triple slash on the line directly above a variable or proc is the correct way to leave comments and is something that the codebase could greatly benefit from having enforced.

Doing this enables you to hover over that variable anywhere in the project to see what it is for, please do not take it away:
![image](https://user-images.githubusercontent.com/9547572/224489433-331780b6-daaf-4ed9-be6c-4a10ab21cb3a.png)

Reply:
Thought VSC was smarter, fixed